lib/utils.js

```diff
@@ -190,38 +167,11 @@ const unsetConfig = (options, config) => {
   }
 }
 
-// Update LogDNA Agent if Enabled
-const updateSelf = () => {
-  if (os.platform() === 'linux') {
-    return spawn('/bin/bash', [
-      '-c'
-    , 'if [[ ! -z $(which apt-get) ]]; then apt-get update; apt-get install -y --force-yes logdna-agent; '
-        + 'elif [[ ! -z $(which yum) ]]; then yum clean expire-cache; yum -y install logdna-agent; elif [[ ! -z $(which zypper) ]]; '
-        + 'then zypper refresh; zypper install -y logdna-agent; fi; sleep 1; /etc/init.d/logdna-agent start'
-    ], {
-      detached: true
-    })
-  } else if (os.platform() === 'win32') {
-    return spawn('cmd.exe', [
-      '/c'
-    , 'choco'
-    , 'upgrade'
-    , 'logdna-agent'
-    , '-y'
-    , '--allowunofficial'
-    ], {
-      detached: true
-    })
-  }
-}
-
 exports.appender = appender
 exports.buildUrl = buildUrl
```

Comment:
I think no need for this method anymore since we are using node client

Reply:
The node client expects a full URL, and since we have it broken up by host, port in the env, we have to build the string.  I'd rather leave it for now, but I'd like to just make this the full URL eventually.  The host and port were needed for the healthcheck now, and that's gone.  Should we tweak it now as part of this major?  maybe....

Reply:
It will deprecate some env variables, right? The env variables which are frequently used by customers - we can easily include that in one of our upcoming major releases but we should clearly state what's been changed - in very noticeable way

Reply:
Lemme see about cleaning that up now.  I also forgot to take the healthcheck constant out.

Reply:
I ripped off the bandaid and make it a full URL.  This removes even more code!

Reply:
BTW, the changes to the env vars, I'd assumed would all go in the CHANGELOG